docs/src/components/InlineFilter.tsx

```diff
@@ -0,0 +1,21 @@
+import { Fragment } from 'react';
+import { FilterChildren } from './FilterChildren';
+
+type InlineFilterProps = {
+  children: React.ReactNode;
+  filters: string[];
+};
+
+export const InlineFilter = ({ filters, children }: InlineFilterProps) => {
```

Comment:
Can we add a comment on this component to what it is doing and how to use it? I have a feeling in 2 years we are going to be like "wtf is this thing doing"